DESCRIPTION.md

```diff
@@ -8,8 +8,13 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 
 # Release Notes
 
-- v2.7.13 (Unreleased)
+- v2.7.13(September 22,2022)
```

Comment:
@sfc-gh-kdama should we update this date?

Reply:
@sfc-gh-sfan let's minor bump because of BCR?
```suggestion
- v2.8.0(September 22,2022)
```

Reply:
Yeah sounds good.